components/mpas-ocean/src/Registry.xml

```diff
@@ -2068,6 +2204,18 @@
 		<var name="ssh" type="real" dimensions="nCells Time" units="m"
 			 description="sea surface height"
 		/>
+                <var name="verticalVelocity" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-1}"
+       		         description="prognostic vertical velocity, only used when config_enable_nonhydrostatic_mode = .true."
+    		/>
+                <var name="nonhydrostaticPressure" type="real" dimensions="nVertLevels nCells Time" units="N m^{-2}"
+       	                 description="nonHydrostatic pressure , set to zero if config_enable_nonhydrostatic_mode = .false."
+                />
+                <var name="tke" type="real" dimensions="nVertLevels nCells Time" units="m^2 s^{-2}"
+                         description="Turbulence Kinetic Energy for the two equation turbulence closure"
+                />
+                <var name="glsPsi" type="real" dimensions="nVertLevels nCells Time" units="variable"
+                         description="psi variable for the two equation turbulence closure (see Umlauf and Burchard 2003).  The units here depend on the choice of variables in the GLS scheme"
+                />
```

Comment:
Here and below in the registry, it would be best not to introduce new variables that aren't used when not in nonhydro mode because of the extra memory this incurs.  If at all possible, these should all be in the `nonhydrostaticPKG`.  So far, you have created the package but not added any variables to it.